modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -646,6 +686,24 @@ export default class AvgRTPStatsReporter {
                     frameRate, true /* local */, VideoType.DESKTOP));
         }
 
+        if (resolution) {
+            this._avgRemoteCameraPixels.addNext(
```

Comment:
It's computing an average over all remote participants. We're going to use it for drawing conclusions in 1:1 calls (so it doesn't really matter whether or not the average is a good aggregator function or not).